pkg/cmd/init.go

```diff
@@ -1,4 +1,4 @@
-package commands
+package cmd
```

Comment:
init needs to be a function for generating the folder structure but an application will not be able to use this if they have their own genesis. can we abstract this into a helper to generate the folder structure and configs but then allow the application to also write their genesis. If the app dev needs to do a command that wraps another command it starts to be a code smell